ramp-tps/src/utils.rs

```diff
@@ -0,0 +1,77 @@
+use bzip2::bufread::BzDecoder;
+use solana_sdk::genesis_block::GenesisBlock;
+use solana_sdk::timing::duration_as_ms;
+use std::fs::File;
+use std::io;
+use std::net::SocketAddr;
+use std::path::Path;
+use std::thread::sleep;
+use std::time::Duration;
+use std::time::Instant;
+use tar::Archive;
+
+const GENESIS_ARCHIVE_NAME: &str = "genesis.tar.bz2";
+
+/// Inspired by solana_local_cluster::cluster_tests
+pub fn sleep_n_slots(num_slots: u64, genesis_block: &GenesisBlock) {
```

Comment:
It would be nice to have tests on this math. Maybe split out the secs calculation into a separate function to unit test?